base/server/src/com/netscape/cms/servlet/common/CMSTemplate.java

```diff
@@ -333,7 +335,7 @@ private String renderValue(Object v) {
                 ((Character) v).equals(Character.valueOf((char) 0))) {
             s = "null";
         } else {
-            s = "\"" + v.toString() + "\"";
+            s = "\"" + CMSTemplate.escapeJavaScriptString(v.toString()) + "\"";
```

Comment:
Question: Why are we using the method in `CMSTemplate` rather than using the method from `StringEscapeUtils` directly? IMO, the method in `CMSTemplate` must be deprecated now and, once all the code that relies on this method is updated we need to remove it. Thoughts?

Reply:
I think the end goal would be to remove `CMSTemplate` entirely in RHEL 8.4 by migrating to Patternfly. We're just keeping the existing thing alive until then.